mimesniff/mime-types/resources/mime-types.json

```diff
@@ -193,6 +193,23 @@
     "input": "x/x;test=\"\\",
     "output": "x/x;test=\"\\\\\""
   },
+  "Whitespace (not handled by generated-mime-types.json)",
+  {
+    "input": "x/x;x= ",
+    "output": "x/x"
+  },
+  {
+    "input": "x/x;\" ",
+    "output": "x/x;\" \""
+  },
+  {
+    "input": "x/x;x=\t",
+    "output": "x/x"
+  },
+  {
+    "input": "x/x;x=\"\t",
+    "output": "x/x;x=\"\t\""
```

Comment:
In my parser the output is `x/x`, with no parameters. Remember that you remove leading and trailing whitespace from the whole MIME type in step 1 of the entire parser algorithm, so this is equivalent to parsing `x/x;x="`